src/addons/editors/AutoCompleteTokensEditor.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import Select from 'react-select';
+import ReactDOM from 'react-dom';
+
+const ExcelColumn = {
+  name: React.PropTypes.string.isRequired,
+  key: React.PropTypes.string.isRequired,
+  width: React.PropTypes.number.isRequired,
+  filterable: React.PropTypes.bool
+};
+
+class AutoCompleteTokensEditor extends Component {
+
+  static propTypes = {
+    options: React.PropTypes.array.isRequired,
+    column: React.PropTypes.shape(ExcelColumn),
+    value: React.PropTypes.array
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: []
+    };
+    this.handleSelectChange = this.handleSelectChange.bind(this);
+  }
+
+  getInputNode() {
+    return ReactDOM.findDOMNode(this);
+  }
+
+  getValue() {
+    let updated = {};
+    updated[this.props.column.key] = this.state.value;
+    return updated;
+  }
+
+  handleSelectChange(value) {
+    this.setState({value});
+  }
+
```

Comment:
I realise this is not documented anywhere yet, but a select based autocomplete editor needs to implement a `hasResults` method similar to https://github.com/adazzle/react-data-grid/blob/master/src/addons/editors/AutoCompleteEditor.js#L66

so that the editorContainer can handle keyboard navigation correctly. (eg pressing down button will focus a select option instead of navigating down one cell if the Select component has results)